src/components/nav_drawer/nav_drawer.tsx

```diff
@@ -124,14 +191,14 @@ export class EuiNavDrawer extends Component {
 
     // Scrolls the menu and flyout back to top when the nav drawer collapses
     setTimeout(() => {
-      document.getElementById('navDrawerMenu').scrollTop = 0;
+      document.getElementById('navDrawerMenu')!.scrollTop = 0;
```

Comment:
While we're here, this file previously defines `const MENU_ELEMENT_ID = 'navDrawerMenu';`

```suggestion
      document.getElementById(MENU_ELEMENT_ID)!.scrollTop = 0;
```